spec/models/tag_spec.rb

```diff
@@ -104,15 +104,9 @@
     end
 
     it "tag with nil classification" do
-      @tag.classification = nil
-      categorization = @tag.categorization
-
-      expected_categorization = {"name"         => nil,
-                                 "description"  => nil,
-                                 "category"     => {"name" => @category.name, "description" => @category.description},
-                                 "display_name" => "#{@category.description}: "}
-      expect(@tag.show).to eq(true)
-      expect(categorization).to eq(expected_categorization)
+      @tag.classification.delete
+      expect(@tag.show).to be_falsey
+      expect(@tag.categorization).to eq({})
```

Comment:
This will fix rails 5.1 and tests and the change to use `classification.delete` will also cause `.show` to be falsey so it feels like the right change... with that said, does this behavior satisfy the BZ this test was originally designed to test?  I think they just wanted the api categorization to not blow up, which would be what the change and this test clearly shows.  Right?

Reply:
Yeah, I think so. With my code changes adding a bazillion `.try`s everywhere and then this corrected test, we should be golden.